gap/attributes/isomorph.gi

```diff
@@ -107,6 +107,69 @@ InstallMethod(IsIsomorphicSemigroup, "for semigroups",
 [IsSemigroup, IsSemigroup],
 {S, T} -> IsomorphismSemigroups(S, T) <> fail);
 
+InstallMethod(IsIsomorphicSemigroup, "for finite simple semigroups",
+[IsSimpleSemigroup and IsFinite, IsSimpleSemigroup and IsFinite],
+function(R, S)
+  local uR, uS, map, mat, next, row, entry, isoR, rmsR, isoS, rmsS;
+
+  # Take an isomorphism of R to an RMS if appropriate
+  if not (IsReesMatrixSemigroup(R) and IsWholeFamily(R)
+      and IsPermGroup(UnderlyingSemigroup(R))) then
+    isoR := IsomorphismReesMatrixSemigroupOverPermGroup(R);
+    rmsR := Range(isoR);
+  else
+    rmsR := R;
+  fi;
+  # Take an isomorphism of S to an RMS if appropriate
+  if not (IsReesMatrixSemigroup(S) and IsWholeFamily(S)
+      and IsPermGroup(UnderlyingSemigroup(S))) then
+    isoS := IsomorphismReesMatrixSemigroupOverPermGroup(S);
+    rmsS := Range(isoS);
+  else
+    rmsS := S;
+  fi;
+
+  rmsR := CanonicalReesMatrixSemigroup(rmsR);
```

Comment:
Is there a reason for computing `CanonicalReesMatrixSemigroup(rmsR);` here instead of later on when we compute `CanonicalReesMatrixSemigroup(rmsS);`? It might be worth testing out if it still works/is faster with the line moved.